packages/eas-cli/src/graphql/client.ts

```diff
@@ -19,6 +23,16 @@ type SessionHeaders = {
 
 export const graphqlClient = createUrqlClient({
   url: getExpoApiBaseUrl() + '/--/graphql',
+  exchanges: [
+    dedupExchange,
+    cacheExchange,
+    retryExchange({
+      maxDelayMs: 4000,
```

Comment:
The default options for `retryExchange` are:
```
{
  initialDelayMs: 1000,
  maxDelayMs: 15000,
  randomDelay: true,
  maxNumberAttempts: 2,
  retryIf: err => err && err.networkError,
};
```

The potential 31 = 2*15+1 second delay seemed excessive. Went with a max potential 9 = 2*4 +1 second delay.

The randomization of the delay and the high max is for dealint with https://en.wikipedia.org/wiki/Thundering_herd_problem which does not seem like it will be as large of a problem when dealing with CLI calls.